galaxy_api/auth/models.py

```diff
@@ -1,5 +1,34 @@
+from django.db import models
 from django.contrib.auth.models import AbstractUser
 
+from galaxy_api.common import models as common_models
+
+
+__all__ = (
+    'User',
+    'Tenant',
+)
+
 
 class User(AbstractUser):
+    """Custom user model."""
     pass
```

Comment:
Not that its a problem but `pass` is unnecessary if there is a docstring.

Reply:
We discussed that once. It was not always possible and it's my personal preference, IMO not having pass can be confusing for those who read the code.

Reply:
yeah, you're right, i remember now. i'll try to not repeat myself, sorry.